awx/ui/urls.py

```diff
@@ -27,4 +31,4 @@ def get_context_data(self, **kwargs):
 
 app_name = 'ui'
 
-urlpatterns = [re_path(r'^$', IndexView.as_view(), name='index'), re_path(r'^migrations_notran/$', MigrationsNotran.as_view(), name='migrations_notran')]
+urlpatterns = [re_path(r'^$', IndexView.as_view(), name='index'), re_path(r'^controller/.*$', ControllerView.as_view(), name='index'), re_path(r'^migrations_notran/$', MigrationsNotran.as_view(), name='migrations_notran')]
```

Comment:
Note the regex pattern `r'^controller/.*$'` here. We need this in order so that hard refreshes work. Otherwise django views(`/controller/projects/?sort....`) as invalid routes